src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+@use "./stars.scss";
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  box-sizing: border-box;
+  font-family: "Roboto", Helvetica, sans-serif;
+  width: 200px;
+  height: 408px;
```

Comment:
Setting fixed width is not the best idea, it would be better to set it to 100%, and sex max-width, so it can stay responsive and shrink if needed. Setting fixed height is really bad idea. What happens if the text wraps? The content will now overflow the container. Height should be determined by contents.

Reply:
updated in 'solution2'